Tasks/HelmDeployV0/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 0,
         "Minor": 178,
```

Comment:
Could you please update the Minor to 179, and Patch to 0 as this is a new sprint.

Reply:
Version was bumped minor to 179 and patch to 0